app/ui-message/client/message.js

```diff
@@ -197,18 +197,11 @@ Template.message.helpers({
 		}
 	},
 	showTranslated() {
-		if (settings.get('AutoTranslate_Enabled') && this.u && this.u._id !== Meteor.userId() && !MessageTypes.isSystemMessage(this)) {
-			const subscription = Subscriptions.findOne({
-				rid: this.rid,
-				'u._id': Meteor.userId(),
-			}, {
-				fields: {
-					autoTranslate: 1,
-					autoTranslateLanguage: 1,
-				},
-			});
-			const language = AutoTranslate.getLanguage(this.rid);
-			return this.autoTranslateFetching || (subscription && subscription.autoTranslate !== this.autoTranslateShowInverse && this.translations && this.translations[language]);
+		const { msg, subscription, settings } = this;
```

Comment:
I guess this is not supposed to be in this PR as well, there is no such properties in `this` object.